### PR TITLE
AvatarGroup Component

### PR DESCRIPTION
**Pull request by jasonleyser**
Created an AvatarGroup component

### CODE REVIEW

File: components/system/components/AvatarGroup.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import * as React from "react";
+import * as Constants from "~/common/constants";
+
+import { css } from "@emotion/react";
+
+const STYLES_UL = css`
+  list-style-type: none;
+
+  :hover li:nth-child(n + 2) {
```
Comment (jimmylee) [inline components/system/components/AvatarGroup.js:9]: What do you need this for?

Comment (jasonleyser) [reply components/system/components/AvatarGroup.js:9]: do you mean the `list-style-type`?

Comment (jimmylee) [reply components/system/components/AvatarGroup.js:9]: No I mean this :hover li:nth-child(n + 2) {}

Why isn't the hover rule inside of the `li` element?

Comment (jasonleyser) [reply components/system/components/AvatarGroup.js:9]: the transition animation doesn't work when I move it to the `li` element

Comment (jimmylee) [reply components/system/components/AvatarGroup.js:9]: Gotcha, we can refine this as we use it. Thanks ^.^

Comment (jasonleyser) [reply components/system/components/AvatarGroup.js:9]: sounds good!

**Pull request merged**